File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex21.java
//Importando Scanner
import java.util.Scanner;

public class Ex21 {
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
	
	//vars
	double horasTrabalhadas, valorSalMinimo, horasExtrasTrabalhadas, salario;
		
	
	System.out.println("Salário a receber\n\n");
	
	//Entrada	
	System.out.print("Insira o quantidade de horas trabalhadas: ");
	horasTrabalhadas = ent.nextDouble();
	
	System.out.print("Insira a quantidade de horas extras trablhadas: ");
	horasExtrasTrabalhadas = ent.nextDouble();
	
	System.out.print("Insira o valor do salário mínimo: ");
	valorSalMinimo = ent.nextDouble();
	
	//Processamento	
	salario = (horasTrabalhadas * valorSalMinimo/8) + (horasExtrasTrabalhadas * valorSalMinimo/4);  
	
	//Saída
	System.out.println("Salário a receber: R$" + salario);
	
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex21.c
#include <stdio.h>

main()
{
    //vars
    float hTrabalhadas, eTrabalhadas, valorHTrabalhadas, valorETrabalhadas, salarioMinimo, salarioFinal;

    printf("\t\t\tSalario a receber\n\n");

    //Entrada;
    printf("Insira a quantidade de horas trabalhadas: ");
    scanf("%f",&hTrabalhadas);

    printf("Insira a quantidade de horas extra trabalhadas: ");
    scanf("%f",&eTrabalhadas);

    printf("Insira o valor do salario minimo: ");
    scanf("%f",&salarioMinimo);

    //Processamento
    valorHTrabalhadas = salarioMinimo / 8;
    valorETrabalhadas = salarioMinimo / 4;
    salarioFinal = (valorHTrabalhadas * hTrabalhadas) + ( valorETrabalhadas * eTrabalhadas );

    //Saída
    printf("Salario a receber: R$%.2f", salarioFinal);

}

